lib/engine/game/g_18_ny/entities.rb

```diff
@@ -351,7 +351,7 @@ module Entities
             always_market_price: true,
             float_percent: 20,
             coordinates: 'F26',
-            color: '#A7A9AC',
+            color: 'red',
```

Comment:
Please use the hex code to stay consistent with the rest of the implementation. The B&A logo uses #E21F27. Apply to the SVGs too.

Reply:
ok